src/backend/executor/execMain.c

```diff
@@ -266,6 +273,42 @@ CopyDirectDispatchFromPlanToSliceTable(PlannedStmt *stmt, EState *estate)
 	CopyDirectDispatchFromPlanToSliceTableWalker((Node *) stmt->planTree, &context);
 }
 
+static inline void
```

Comment:
Why do we need `inline` here and below? Compiler can decide for itself whether to inline the function or not.

Reply:
Adding `inline` was suggested by Georgy, and as it is only a hint for the compiler, I consider having it is not worse than not having. So it was added.